db/seeds.rb

```diff
@@ -92,7 +93,7 @@
 puts 'Updating permissions'
 admin = Group.find_by name: 'admin'
 Permission.update_permissions_table
-Permission.create(controller: 'rentals', action: 'cost_adjustment')
+Permission.create(controller: 'rentals', action: 'cost_adjustment') # this is a special permission that isnt really a controller action
```

Comment:
So you have a controller action that's completely unroutable but that serves as a placeholder for whether or not someone has the permission to do something on a different page?